src/runtime/pkg/resourcecontrol/utils_linux.go

```diff
@@ -43,28 +45,36 @@ func ValidCgroupPathV1(path string, systemdCgroup bool) (string, error) {
 	return filepath.Join(DefaultResourceControllerID, filepath.Clean("/"+path)), nil
 }
 
-// ValidCgroupPathV2 returns a valid cgroup path for cgroup v2.
-// see https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#cgroups-path
-func ValidCgroupPathV2(path string, systemdCgroup bool) (string, error) {
+func absolutePath(path string) string {
 	// In cgroup v2，path must be a "clean" absolute path starts with "/".
-	if IsSystemdCgroup(path) {
-		return filepath.Join("/", path), nil
-	}
-
-	if systemdCgroup {
-		return "", fmt.Errorf("malformed systemd path '%v': expected to be of form 'slice:prefix:name'", path)
-	}
+	return filepath.Join("/", path)
+}
 
-	// In the case of an absolute path (starting with /), the runtime MUST
-	// take the path to be relative to the cgroups mount point.
-	if filepath.IsAbs(path) {
-		return filepath.Clean(path), nil
+func RandStr(n int) string {
```

Comment:
Why do you need to invent a function to generate a random string? Other places do something like:
```
	sandboxId := uuid.Generate().String()
```

Wouldn't that be enough?

Reply:
Sorry, I don't know a better way, I have replaced it.